src/pdmat.jl

```diff
@@ -84,8 +84,8 @@ end
 Base.inv(a::PDMat) = PDMat(inv(a.chol))
 LinearAlgebra.det(a::PDMat) = det(a.chol)
 LinearAlgebra.logdet(a::PDMat) = logdet(a.chol)
-LinearAlgebra.eigmax(a::PDMat) = eigmax(a.mat)
-LinearAlgebra.eigmin(a::PDMat) = eigmin(a.mat)
+LinearAlgebra.eigmax(a::PDMat) = eigmax(Hermitian(a.mat))
+LinearAlgebra.eigmin(a::PDMat) = eigmin(Hermitian(a.mat))
```

Comment:
We only support `Real` matrices, so `Symmetric` is more appropriate?
```suggestion
LinearAlgebra.eigmax(a::PDMat) = eigmax(Symmetric(a.mat))
LinearAlgebra.eigmin(a::PDMat) = eigmin(Symmetric(a.mat))
```

Reply:
Makes sense